adafruit_l3gd20.py

```diff
@@ -212,12 +220,14 @@ class L3GD20_I2C(L3GD20):
     gyro_raw = Struct(_L3GD20_REGISTER_OUT_X_L_X80, "<hhh")
     """Gives the raw gyro readings, in units of rad/s."""
 
-    def __init__(self, i2c, rng=L3DS20_RANGE_250DPS, address=0x6B):
+    def __init__(
+        self, i2c, rng=L3DS20_RANGE_250DPS, rate=L3DS20_RATE_100HZ, address=0x6B
```

Comment:
Re-order rate here for the same reason. Please try and apply my comments to all places you think it applies.

Reply:
Will do. Sorry about that.